source/frontend/src/features/mapSideBar/router/FilePropertyRouter.tsx

```diff
@@ -37,6 +37,7 @@ export const FilePropertyRouter: React.FC<IFilePropertyRouterProps> = props => {
   const onChildSuccess = () => {
     props.setIsEditing(false);
     setStaleLastUpdatedBy(true);
+    props.onSuccess();
```

Comment:
what is this for?

Reply:
once the take is updated, the need to re-fetch the properties is bubbled up though the onSuccess. Otherwise the map still shows the old markers